packages/injective-protobuf/build.rs

```diff
@@ -9,6 +9,7 @@ fn main() {
 
     protobuf_codegen_pure::Codegen::new()
         .out_dir("src/proto")
+        // TODO: Add more inputs as needed
```

Comment:
like what?

Reply:
Not sure. Others (@ewoolsey?) at some point required more generations here. Perhaps a user of this can come up with a good list to add here.